addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -328,6 +328,125 @@ options.registry.sizing_x = options.registry.sizing.extend({
     },
 });
 
+options.registry.layout_column = options.Class.extend({
+    /**
+     * @override
+     */
+    start: function () {
+        this.$overlay.find('.o_handle.o_add_column, .o_handle.o_remove_column')
+            .removeClass('readonly')
+            .on('click', this._onButtonClick.bind(this));
+        return this._super.apply(this, arguments);
+    },
+    /**
+     * @override
+     */
+    onFocus: function () {
+        this._toggleButtons();
+    },
+
+    //--------------------------------------------------------------------------
+    // Private
+    //--------------------------------------------------------------------------
+
+    /**
+     * Add new column which is clone of last column
+     *
+     * @private
+     */
+    _addColumn: function () {
+        this.trigger_up('request_history_undo_record', {$target: this.$target});
+        var $lastColumn = this._getColumns().last();
+        $lastColumn.clone().insertAfter($lastColumn);
+        this._updateSnippet();
+    },
+    /**
+     * Return all the columns of currently selected snippet.
+     *
+     * @private
+     * @returns {jQuery}
+     */
+    _getColumns: function () {
+        return this.$target.find('.container > .row > [class*="col-md-"]');
+    },
+    /**
+     * Remove last column
+     *
+     * @private
+     */
+    _removeColumn: function () {
+        this.trigger_up('request_history_undo_record', {$target: this.$target});
+        var $lastColumn = this._getColumns().last();
+        $lastColumn.remove();
+        this._updateSnippet();
+    },
+    /**
+     * Resize the columns
+     *
+     * @private
+     */
+    _resizeColumns: function () {
+        var $columns = this._getColumns();
+        var colsLength = $columns.length;
+        if (colsLength <= 6) {
+            var colSize = Math.round(12 / colsLength);
+            $columns.each(function () {
+                $(this).attr('class', $(this).attr('class').replace(/\s*(col-md-|col-md-offset-)([0-9-]+)/g, ''));
+                $(this).addClass('col-md-' + colSize);
```

Comment:
Use `_.each` instead of `$.each` (and adapt to not build the `$(this)` jQuery object multiple times).